.github/scripts/Release.py

```diff
@@ -20,7 +20,7 @@
 
 VERSION_PATTERN = r'(\d{4})\.(\d{2})\.(\d{2})(\.\d+)?'
 PUBLICATION_DATE_PATTERN = r'\d{4}-\d{2}-\d{2}'
-DOI_PATTERN = r'10.\d{4,9}/[-._;()/:a-zA-Z0-9]+'
+DOI_PATTERN = r'10\.\d{4,9}/zenodo\.(\d+)'
```

Comment:
Why the parens?

Reply:
Copied from `CheckMetadata.py` 🤷‍♂️ want me to remove them?

Reply:
removed while squashing